src/pages/New-SignUp/NewSignUp.tsx

```diff
@@ -4,7 +4,7 @@ import useNavigateForms from "../../hooks/useNavigateForms";
 import { TNewUserFields } from "../../types/type";
 import SubmitSuccess from "./SubmitSuccess";
 import { useState } from "react";
-
+// 
```

Comment:
Is this intentional?

Reply:
yeah. i just needed a change to push. i have edited it.
the pipeline is failing with this error - Module not found: Error: Can't resolve '../../hooks/useNavigateForms' in '/app/src/pages/New-SignUp'